### ISSUE TITLE
Check if module is imported twice.

### ISSUE THREAD

**Issue by anntzer** [PyCQA/pycodestyle#900]
There is currently no warning emitted e.g. for
```python
import io
import io
```
but having one would be nice.
(A more realistic case would be a long, unsorted list of imports where the same import occurs twice relatively far away from one another.)

**Comment by FichteFoll** [comment]
Afaik, pycodestyle doesn't do lexical analysis to this degree, but flake8 most likely does, since it annotated redefinitions and unused imports or variables.

**Comment by anntzer** [comment]
Oopsies, I agree this belongs to pyflakes (which does check for this via F811), not pycodestyle.
